shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -273,6 +312,17 @@ module MakeImpl<InputSig Lang> {
       not fullBarrier(node2)
     }
 
+    /** Provides the relevant barriers for a step from `node1,state1` to `node2,state2`, including stateless barriers for `node1` to `node2`. */
+    bindingset[node1, state1, node2, state2]
+    pragma[inline]
```

Comment:
Not needed.

Reply:
You mean you want `pragma[inline]` removed because it is implied by `bindingset`?

Reply:
Yes.

Reply:
Done